### DISCUSSION TITLE
[Esame 9 Gennaio 2024] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#429] [Esami - Viaggi/Piazza] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/00800743-49eb-4ab9-b99d-78cbaa7edb56)

**Comment by Elia-Belli** [comment, marked as answer]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/554c906f-c0fe-4926-8c08-b592a233af0d)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-14 233008](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/1faf915b-5e71-4e5d-b28f-fb075df77fcd)
![Immagine 2024-01-14 233020](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/c34d3b17-8f44-4f3a-a132-174914b632b9)
![Immagine 2024-01-14 233036](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/25f9da19-ae4b-401b-8b8d-d64c45c66131)
![Immagine 2024-01-14 233045](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/da166d74-f85d-469e-b865-f5fa37462f79)
